org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/WorkDoneProgressKind.java

```diff
@@ -0,0 +1,43 @@
+/******************************************************************************
+ * Copyright (c) 2016 TypeFox and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ ******************************************************************************/
+package org.eclipse.lsp4j;
+
+public enum WorkDoneProgressKind {
+	BEGIN("begin"),
```

Comment:
Maybe it's worthwhile to use lowercase enum literals here which would render basically the entire logic in this enum declaration obsolete.
´public enum WorkDoneProgressKind { begin, report, end; }`

Reply:
ok for me, definitly not used to use lower on enum, but why not